crates/csharp/src/lib.rs

```diff
@@ -601,7 +608,14 @@ impl InterfaceGenerator<'_> {
                 r#"
                 private unsafe struct ReturnArea
                 {{
-                    private int GetS32(IntPtr ptr, int offset)
+                    internal int GetS32(IntPtr ptr, int offset)
+                    {{
+                        var span = new Span<byte>((void*)ptr, {});
+
+                        return BitConverter.ToInt32(span.Slice(offset, 4));
+                    }}
+
+                    internal int GetF32(IntPtr ptr, int offset)
```

Comment:
Should this be returning a float ?

Reply:
Pushed a fix

Reply:
Thanks!  I haven't had a chance to get back to this but will try to in the next couple of days